pxr/imaging/plugin/hioImageIO/plugInfo.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Plugins": [
+    {
+      "Info": {
+        "Types": {
+          "HioImageIO_Image" : {
+            "bases": ["HioImage"],
+            "imageTypes": ["tif", "tiff", "zfile", "tx"],
+            "precedence": 10
```

Comment:
If we build both OpenImageIO and ImageIO, we choose the plugin to use based on precedence.  Since hioOiio is "10", we could bump this to like "15" and then it would be used preferentially.